priority_variables_transform/CHS-ingest/cause_of_death.yaml

```diff
@@ -22,6 +22,8 @@
                       '4': ICD10CM:I00-I99
                   order:
                     value: 1
+        age_at_death:
+          expr: case({phv00100774} == 1, ({pht001452.phv00100487} * 365) + {phv00100776})
```

Comment:
parens and internal case again:
```
expr: case(({phv00100774} == 1, case(({pht001452.phv00100487} * 365) + {phv00100776})))
```
Oh, and we have that funny `pht#.phv#` construction again that we can't handle yet.